pkg/component/status/status.go

```diff
@@ -47,6 +55,21 @@ func (s *Status) Init() error {
 	return nil
 }
 
+// checkLeftoverSocket checks if there is a leftover socket from the previous runs. This usually happens on crash or SIGKILL
+func checkLeftoverSocket(socket string) error {
+	_, err := net.Dial("unix", "socket")
```

Comment:
Shouldn't this be the variable `socket` instead of string literal `"socket"`?

Reply:
Yes, of course! Thank you!